oap-server/server-starter/src/main/resources/ui-initialized-templates/apisix/apisix-service.json

```diff
@@ -0,0 +1,646 @@
+            {
+              "widgetId": "1"
+            },
+            {
+              "widgetId": "2"
+            },
+            {
+              "widgetId": "3"
+            },
+            {
+              "widgetId": "4"
+            },
+            {
+              "widgetId": "6"
+            },
+            {
+              "widgetId": "7"
+            },
+            {
+              "widgetId": "8"
+            }
+          ]
+        },
+
+        {
+          "x": 6,
+          "y": 26,
+          "w": 6,
+          "h": 13,
+          "i": "10",
+          "type": "Widget",
+          "widget": {
+            "title": "Shared Dict Free Space (MB)"
+          },
+          "graph": {
+            "type": "Line",
+            "step": false,
+            "smooth": false,
+            "showSymbol": false,
+            "showXAxis": true,
+            "showYAxis": true
+          },
+          "metrics": [
+            "meter_apisix_shared_dict_free_space_bytes"
+          ],
+          "metricTypes": [
+            "readLabeledMetricsValues"
+          ],
+          "metricConfig": [
+            {
+              "calculation": "byteToMB"
+            }
+          ],
+          "associate": [
+            {
+              "widgetId": "1"
+            },
+            {
+              "widgetId": "2"
+            },
+            {
+              "widgetId": "3"
+            },
+            {
+              "widgetId": "4"
+            },
+            {
+              "widgetId": "6"
+            },
+            {
+              "widgetId": "7"
+            },
+            {
+              "widgetId": "8"
+            }
+          ]
+        },
+        {
+          "x": 12,
+          "y": 26,
+          "w": 6,
+          "h": 13,
+          "i": "11",
+          "type": "Widget",
+          "widget": {
+            "title": "Etcd indexes"
+          },
+          "graph": {
+            "type": "Line",
+            "step": false,
+            "smooth": false,
+            "showSymbol": false,
+            "showXAxis": true,
+            "showYAxis": true
+          },
+          "metrics": [
+            "meter_apisix_sv_etcd_indexes"
+          ],
+          "metricTypes": [
+            "readLabeledMetricsValues"
+          ],
+          "associate": [
+            {
+              "widgetId": "1"
+            },
+            {
+              "widgetId": "2"
+            },
+            {
+              "widgetId": "3"
+            },
+            {
+              "widgetId": "4"
+            },
+            {
+              "widgetId": "6"
+            },
+            {
+              "widgetId": "7"
+            },
+            {
+              "widgetId": "8"
+            }
+          ]
+        },
+
+        {
+          "x": 18,
+          "y": 26,
+          "w": 6,
+          "h": 13,
+          "i": "14",
+          "type": "Widget",
+          "widget": {
+            "title": "Etcd Reachable (0/1)"
```

Comment:
```suggestion
            "title": "etcd latest reachability"
```
Please add `0 represents etcd unreachable, 1 represents etcd connected` in the tip
<img width="525" alt="image" src="https://user-images.githubusercontent.com/5441976/193851240-4174e6d8-e989-434d-99d0-dfbc4fa07713.png">

Reply:
<img width="524" alt="image" src="https://user-images.githubusercontent.com/3917424/193860147-8800a62b-ffe6-435e-a772-616f673f3da8.png">

Reply:
BTW , is there any way to convert double to int in UI ?

Reply:
I think we don't have that. For your case, an expression is better than that.

Reply:
yes ， expression is  better . is it supported ?

Reply:
No, it is just an idea, I am concerned it is too complex.